packages/wrangler/src/images/local.ts

```diff
@@ -0,0 +1,187 @@
+import { File } from "buffer";
+import sharp from "sharp";
+import { z } from "zod";
+import type { ImageInfoResponse } from "@cloudflare/workers-types/experimental";
+import type { Sharp } from "sharp";
+
+const Transform = z.object({
+	imageIndex: z.number().optional(),
+	rotate: z.number().optional(),
+	width: z.number().optional(),
+	height: z.number().optional(),
+});
+
+const Transforms = z.array(Transform);
+
+export async function imagesLocalFetcher(request: Request): Promise<Response> {
+	const data = await request.formData();
+
+	const body = data.get("image");
+	if (!body || !(body instanceof File)) {
+		return errorResponse(400, 9523, "ERROR: Expected image in request");
```

Comment:
can we be a bit more helpful here for our users in our err message. Maybe smth like

```suggestion
		return errorResponse(400, 9523, "ERROR: Expected image in request body but got ....");
```